compiler/parser/Cargo.toml

```diff
@@ -69,3 +74,4 @@ version = "0.8"
 [features]
 default = [ ]
 ci_skip = [ ]
+wasm = [ "snarkvm-console/wasm","dep:getrandom" ]
```

Comment:
Please use the `snarkvm-console` workspace dependency with `wasm` feature flag

Reply:
sry, i dont get what you mean, wasm is an optional feature, it can only be added from here, if i add this feature from workspace, it will be inherited to `leo` as an unnecessary feature.

Reply:
@code-pangolin I believe this will need to be rebased onto `testnet3` and updated appropriately.